docs/modules/operation/pages/performance-data-collection/snmp-property-extenders/snmp-interface.adoc

```diff
@@ -0,0 +1,118 @@
+
+[source, xml]
+----
+<resourceType name="dot1dBasePortEntry" label="dot1d Base Port" resourceLabel="${index}">
+  <persistenceSelectorStrategy class="org.opennms.netmgt.collection.support.PersistAllSelectorStrategy" />
+  <storageStrategy class="org.opennms.netmgt.collection.support.IndexStorageStrategy" />
+</resourceType>
+
+<groups>
+  <group name="ifTable" ifType="all">
+    <mibObj oid=".1.3.6.1.2.1.2.2.1.1"     instance="ifIndex" alias="interfaceIndex" type="string" /> <1>
+    <mibObj oid=".1.3.6.1.2.1.2.2.1.2"     instance="ifIndex" alias="interfaceDescr" type="string" />
+    <mibObj oid=".1.3.6.1.2.1.31.1.1.1.1"  instance="ifIndex" alias="interfaceName"  type="string" />
+    <mibObj oid=".1.3.6.1.2.1.31.1.1.1.18" instance="ifIndex" alias="interfaceAlias" type="string" />
+  </group>
+
+  <group name="dot1dBasePortTable" ifType="all">
+    <mibObj oid=" .1.3.6.1.2.1.17.1.4.1.1" instance="dot1dBasePortEntry" alias="dot1dBasePort"        type="string" />
+    <mibObj oid=" .1.3.6.1.2.1.17.1.4.1.2" instance="dot1dBasePortEntry" alias="dot1dBasePortIfIndex" type="string" /> <2>
+    <mibObj oid=" .1.3.6.1.2.1.17.1.4.1.4" instance="dot1dBasePortEntry" alias="d1dBPDelayExDiscard"  type="counter" />
+    <mibObj oid=" .1.3.6.1.2.1.17.1.4.1.5" instance="dot1dBasePortEntry" alias="d1dBPMtuExDiscard"    type="counter" />
+    <property instance="dot1dBasePortEntry" alias="dot1dBasePortIfDescr" class-name="org.opennms.netmgt.collectd.InterfaceSnmpPropertyExtender"> <3>
+      <parameter key="source-ifindex-attribute" value="interfaceIndex"/>
+      <parameter key="source-attribute" value="interfaceDescr"/> <4>
+      <parameter key="target-ifindex-pointer-column" value="dot1dBasePortIfIndex"/>
+    </property>
+    <property instance="dot1dBasePortEntry" alias="dot1dBasePortIfName" class-name="org.opennms.netmgt.collectd.InterfaceSnmpPropertyExtender"> <5>
+      <parameter key="source-ifindex-attribute" value="interfaceIndex"/>
+      <parameter key="source-attribute" value="interfaceName"/> <6>
+      <parameter key="target-ifindex-pointer-column" value="dot1dBasePortIfIndex"/>
+    </property>
+    <property instance="dot1dBasePortEntry" alias="dot1dBasePortIfAlias" class-name="org.opennms.netmgt.collectd.InterfaceSnmpPropertyExtender"> <7>
+      <parameter key="source-ifindex-attribute" value="interfaceIndex"/>
+      <parameter key="source-attribute" value="interfaceAlias"/> <8>
+      <parameter key="target-ifindex-pointer-column" value="dot1dBasePortIfIndex"/>
+    </property>
+  </group>
+</groups>
+----
+<1> First we collect all of `ifIndex`, `ifDescr`, `ifName`, and `ifAlias` in a group associated with the `ifIndex` source resource-type, using modified names to avoid collisions with internal workings (the `ifIndex` type is built in, so we do not need a custom resource-type definition for it)
```

Comment:
```suggestion
<1> First we collect all of `ifIndex`, `ifDescr`, `ifName`, and `ifAlias` in a group associated with the `ifIndex` source resource-type, using modified names to avoid collisions with internal workings (the `ifIndex` type is built in, so we do not need a custom resource-type definition for it).
```